roles/docker/defaults/main.yml

```diff
@@ -10,3 +10,33 @@ docker_repo_info:
   repos:
 
 docker_dns_servers_strict: yes
+
+docker_container_storage_setup: false
```

Comment:
I'd suggest namespace the vars with the distribution as these are very specific to atomic.